Sources/Build/Command.link(SwiftModule).swift

```diff
@@ -84,6 +93,11 @@ extension Command {
         case .Library(.Dynamic):
             args.append("-emit-library")
         case .Executable:
+            // FIXME: This is inefficient, we can store the main module in enum payload.
```

Comment:
Also, given it is subtle, we should definitely have a comment here explaining exactly why this is important (maybe reference SR if a succinct comment isn't clear enough)